src/main/java/duke/Parser.java

```diff
@@ -0,0 +1,42 @@
+package duke;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+
+/**
+ * duke.Parser class to parse the commands entered during run time
+ */
+
+public class Parser {
+
+    /**
+     * Splits the input line into description and time for the deadline
+     * or event
+     * @param string    the input line
+     * @return          array containing the description and time info
+     */
+    public static String[] splitDescriptionAndTime(String string) {
```

Comment:
Not too sure if I quite like the naming convention for the parameter, string. 
Might I suggest something like "inputString".